index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <a href="index.html">
+        <img src="img/logo.png" alt="판다마켓" />
+      </a>
+      <a class="login_button" href="login.html">로그인</a>
+    </header>
+    <main>
+      <section class="top_section_container">
+        <div class="top_section_container_all">
+          <div class="top_section_text">
+            <p class="top_section_text_title">
+              일상의 모든 물건을<br />거래해보세요
+            </p>
+            <a class="top_section_text_button" href="items.html"
+              >구경하러 가기</a
+            >
```

Comment:
💊 제안
현재 코드에서는 태그 하나가 나눠져 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
            <a class="top_section_text_button" href="items.html">
              구경하러 가기
            </a>
```